assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -135,6 +142,60 @@ export const actions = {
 		const property = properties[ 0 ] || { id: PROPERTY_CREATE };
 		registry.dispatch( STORE_NAME ).selectProperty( property.id );
 	},
+
+	/**
+	 * Creates a new Analytics account.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} args              Argument params.
+	 * @param {string} args.accountName  Google Analytics account name.
+	 * @param {string} args.propertyName Google Analytics property name.
+	 * @param {string} args.profileName  Google Analytics profile name.
+	 * @param {string} args.timezone     Google Analytics timezone.
+	 * @return {Function} Generator function action.
+	 */
+	*createAccount( { accountName, propertyName, profileName, timezone } ) {
+		invariant( accountName, 'accountName is required.' );
+		invariant( propertyName, 'propertyName is required.' );
+		invariant( profileName, 'profileName is required.' );
+		invariant( timezone, 'timezone is required.' );
+
+		let accountTicket;
+		yield {
+			payload: {},
+			type: START_FETCH_CREATE_ACCOUNT,
+		};
+
+		try {
+			accountTicket = yield {
+				payload: { accountName, propertyName, profileName, timezone },
+				type: FETCH_CREATE_ACCOUNT,
+			};
+
+			yield actions.receiveCreateAccount( accountTicket );
+
+			yield {
+				payload: {},
+				type: FINISH_FETCH_CREATE_ACCOUNT,
+			};
+		} catch ( error ) {
+			return {
+				payload: { error },
+				type: CATCH_FETCH_CREATE_ACCOUNT,
+			};
+		}
+		return accountTicket;
```

Comment:
This should preferably return `{ response, error }` instead (with `response` being `accountTicket`) for consistency with the other APIs. `error` would be the error caught above, or `undefined`. See e.g. https://github.com/google/site-kit-wp/blob/a1e01f986e03f2b5dfc1b23ad06e76579cff6192/assets/js/modules/analytics/datastore/accounts.js#L87

Reply:
updated in https://github.com/google/site-kit-wp/pull/1390/commits/5f10856a2893b2ed696a559e4e1066d42a445d90